src/AssetManager.php

```diff
@@ -2,35 +2,86 @@
 
 namespace DigitallyHappy\Assets;
 
+use Exception;
+use Illuminate\Support\Str;
+
 class AssetManager
 {
-    public $loaded;
+    const TYPE_STYLE = 'style';
+    const TYPE_SCRIPT = 'script';
+
+    const STATUS_DISABLED = 'Cache CDN is disabled in the configuration.';
+    const STATUS_LOCAL = 'Asset is not in a CDN.';
+    const STATUS_IN_CACHE = 'Asset was already in cache.';
+    const STATUS_DOWNLOADED = 'Asset downloaded.';
+    const STATUS_NO_ACTION = 'Asset was not downloaded, falling back to CDN.';
+
+    private $loaded;
 
     public function __construct()
     {
         $this->loaded = [];
     }
 
-    public function echoCss($path)
+    /**
+     * Outputs a file depending on its type.
+     *
+     * @param  string  $path
+     * @return void
+     */
+    public function echoFile(string $path, array $attributes = [], string $type = null): void
```

Comment:
When you're echoing multiple bassets one after another, the output is on the same line, which looks odd:

![CleanShot 2023-01-03 at 21 16 26](https://user-images.githubusercontent.com/1032474/210425814-34524070-7aec-4f08-9401-7302bd0e5de2.png)

Instead of making the source code look super-clean, it does the opposite.

Reply:
Fixed in https://github.com/DigitallyHappy/assets/pull/17/files